src/regtech_data_validator/create_schemas.py

```diff
@@ -104,7 +104,7 @@ def validate(schema: DataFrameSchema, submission_df: pd.DataFrame, max_errors: i
     is_valid = True
     findings_df: pd.DataFrame = pd.DataFrame()
     next_finding_no: int = 1
-    single_field = multi_field = register = 0
+    error_counts = warning_counts = 0
```

Comment:
Since we're setting these to an object and not just an integer, syntactically more correct to initialize these to None.  This way should something happen we'd get a more obvious NoneType error instead of something like "integer doesn't have value .single_field_count" or whatever

Reply:
Ignore this, @lchen-2101 has a better idea

Reply:
I think instantiating with a default object instead of `None` might be more preferred, considering we were using `0` as the starting point before; and since we don't have null / None check further down in the code, using defensive programming here makes more sense to me.